### PR TITLE
Port license-header for npm

### PR DESCRIPTION
**Pull request by timostamm**

### CODE REVIEW

**Review by smaye81 — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
+{
+  "name": "@bufbuild/connect-es-utils",
```
Comment (bufdev) [inline package.json]: This needs to be updated

File: npm-packages/license-header/.npmignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+**/*.test.*
```
Comment (bufdev) [inline npm-packages/license-header/.npmignore:1]: Why `npm-packages`?

**Review by bufdev — COMMENTED** [review]
Needs updates to README

File: package.json (hunk 1 above)
Comment (timostamm) [reply package.json]: Good catch, fixed.

File: npm-packages/license-header/.npmignore (hunk 2 above)
Comment (timostamm) [reply npm-packages/license-header/.npmignore:1]: We'd usually use `packages` - went with `npm-packages` here to be explicit. If you prefer, we can stick to `packages`, or go with just `npm`.

Updated README in https://github.com/bufbuild/tools/pull/5/commits/f5372b053aff37d821290076eb64e5378d450125. Seems sufficient to me.

**Review by bufdev — APPROVED** [review]

**Pull request merged**